src/core/data/electrons/electrons.hpp

```diff
@@ -306,6 +320,9 @@ class Electrons : public LayoutHolder<typename Ions::gridlayout_type>
     {
     }
 
+    Electrons(Electrons const&) = default;
+    Electrons(Electrons&&)      = delete;
```

Comment:
move wouldn't do anything as there's no ownership of objects here, we would just end up copying pointers anyway

Reply:
is deleting it, from the code's perspective, not sending the message that having it would not be "valid"? usually I only delete stuff that I know is wrong or bad perf etc. Here it does not strike me to be neither